Sources/EdgeNetworkHandlers/IdentityMap.swift

```diff
@@ -54,6 +56,23 @@ struct IdentityMap {
     func getItemsFor(namespace: String) -> [IdentityItem]? {
         return items[namespace]
     }
+
+    /// Decodes a [String: Any] dictionary into an `IdentityMap`
+    /// - Parameter eventData: the event data representing `IdentityMap`
+    /// - Returns: an `IdentityMap` that is represented in the event data, nil if data is not in the correct format
+    static func from(eventData: [String: Any]) -> IdentityMap? {
```

Comment:
Copied from IdentityEdge.

Reply:
What if we pass the IdentityMap from XDM shared state to Konductor directly as dictionary instead of decoding it as IdentityMap? Edge doesn't need to know the IdentityMap structure anymore as the XDM Shared state should guarantee the correct XDM format based on the API contract.

Reply:
Ok, we can just delete IdentityMap from this repo.